[kubernetes] Add kubernetes getting started doc (#3135)
open-telemetry/opentelemetry.io: content/en/docs/collector/getting-started.md

handling, `Service` configuration based on the OpenTelemetry configuration,
automatic sidecar injection into deployments, among others.

For a comprehensive guide to using the collector with Kubernetes, see

> [Kubernetes Getting Started](../../kubernetes/getting-started).

Reviewer: In general, use relative page links for pages in the same section, not across sections.

```suggestion
[Kubernetes Getting Started](/docs/kubernetes/getting-started/).
```

> For an in-depth walkthrough for using the collector in Kubernetes, see
> [Kubernetes Getting Started](../../kubernetes/getting-started).

Reviewer: ```suggestion
For a comprehensive guide to using the collector with Kubernetes, see
[Kubernetes Getting Started](../../kubernetes/getting-started).
```
